guides/repo-view.md

```diff
@@ -2,17 +2,17 @@
 layout: doc
 sublayout: guide
 title: Working with repositories in the browser
-reviewed: 2015-05-29T15:19:05-04:00
+reviewed: 2017-04-28T15:10:24-04:00
 ---
 
 The repository view provides access to the full information for a given repository. It is divided
-into a number of tabs, each will provides specific information and tools.
+into a number of tabs, each provides specific information and tools.
 
 ### Repository View Tabs
 
 - <i class="fa fa-info-circle tab-icon"></i> Information: A basic overview of the repository, including its current builds and the repository's description
 - <i class="fa fa-tags tab-icon"></i> [Tags](/guides/tag-operations.html): Allows for viewing a repository's tags and performing operations on it.
 - <i class="fa fa-tasks tab-icon"></i> [Builds](/guides/building.html): Allows for viewing and starting repository builds, and defining [Build Triggers](/glossary/build-trigger.html).
-- <i class="fa fa-code-fork tab-icon"></i> Visualize: Shows a tree for visualizing the difference between two or more tags.
```

Comment:
nit: These guys are slightly out of order. It goes information, tags, tag history, builds, logs, and then settings.